crates/ruff_linter/src/rules/flake8_pie/rules/unnecessary_placeholder.rs

```diff
@@ -71,8 +71,8 @@ impl AlwaysFixableViolation for UnnecessaryPlaceholder {
     fn fix_title(&self) -> String {
         let Self { kind } = self;
         match kind {
-            Placeholder::Pass => format!("Remove unnecessary `pass`"),
-            Placeholder::Ellipsis => format!("Remove unnecessary `...`"),
+            Placeholder::Pass => "Remove unnecessary `pass`".to_string(),
+            Placeholder::Ellipsis => "Remove unnecessary `...`".to_string(),
         }
     }
```

Comment:
here you might be able to do something like

```rs
    fn fix_title(&self) -> String {
        let title = match &self.kind {
            Placeholder::Pass => "Remove unnecessary `pass`",
            Placeholder::Ellipsis => "Remove unnecessary `...`",
        };
        title.to_string()
    }
```

Reply:
Thanks, changed this according to your suggesions.